pipelines/perf-eval/Controller/job-scheduling.yml

```
@@ -0,0 +1,66 @@
+trigger: none
+schedules:
+  - cron: "30 1 */2 * *" # Every 2 days at 1:30 AM
+    displayName: "1:30 AM every 2 days"
+    branches:
+      include:
+        - main
+        - vitto/kwok-cl2 # to be removed after the PR is merged
+    always: true
+variables:
+  SCENARIO_TYPE: perf-eval
+  SCENARIO_NAME: job-scheduling
+  SCENARIO_VERSION: main
+stages:
+  - stage: azure_eastus2
+    dependsOn: []
+    jobs:
+      - template: /jobs/competitive-test.yml
+        parameters:
+          cloud: azure
+          regions:
+            - eastus2
+          engine: clusterloader2
+          engine_input:
+            image: "ghcr.io/azure/clusterloader2:v20241022"
+          topology: kwok
+          matrix:
+            default:
+              node_count: 2000 # 2k kwok nodes
+              job_throughput: 800 # qps
+              job_count: 20000
+              cilium_enabled: False
+              scale_timeout: "1h"
+              service_test: False
+              workload_type: "job"
+              cl2_config_file: config.yaml
+          max_parallel: 1
+          timeout_in_minutes: 360
+          credential_type: service_connection
+          ssh_key_enabled: false
+  - stage: aws_eastus2
+    dependsOn: []
+    jobs:
+      - template: /jobs/competitive-test.yml
+        parameters:
+          cloud: aws
+          regions:
+            - us-east-2
+          engine: clusterloader2
+          engine_input:
+            image: "ghcr.io/azure/clusterloader2:v20241022"
```

```suggestion
            image: "ghcr.io/azure/clusterloader2:v20250423"
```
Test your changes with newer image.